src/components/inputs/TextInput.js

```diff
@@ -209,6 +229,38 @@ export default class TextInput extends BaseInput {
     }
   }
 
+  renderTitle() {
+    const {floatingPlaceholder, title} = this.props;
+    if (!floatingPlaceholder && title) {
+      const capitalizedTitle = _.capitalize(title);
+      return (
+        <Text
+          style={this.styles.title}
+          text90
+        >
+          {capitalizedTitle}
+        </Text>
+      );
+    }
+  }
+
+  renderCharCounter() {
+    const {maxLength, characterCounter} = this.props;
+    if (maxLength && characterCounter) {
+      const {value} = this.state || '';
+      const counter = value !== undefined ? value.length : 0;
```

Comment:
Use (lodash) _.size here for cleaner code, it will return 0 for undefined value otherwise will return the string length